server/api/auth/authController.js

```diff
@@ -0,0 +1,35 @@
+var createUser = require('../user/userController').create;
+var passport = require('passport');
+var jwt = require('jsonwebtoken');
+var config = require('../../config/development');
```

Comment:
reference `/config/environment`. That'll grab `index.js`, which will dynamically set the variables based on the environment. (This will work once I bring in my PR...)